Applications/spire/source/charting/cached_chart_model.cpp

```diff
@@ -62,6 +61,9 @@ std::vector<ChartRange> CachedChartModel::get_gaps(ChartValue first,
       range_points.front() = range.m_end;
     } else if(range.m_start <= last && range.m_end >= last) {
       range_points.back() = range.m_start;
+    } else {
+      range_points.insert(range_points.end() - 1, range.m_start);
```

Comment:
Remove this snippet of code, write a unit test that fails, commit that test to git. Reinsert this snippet of code, verify that the unit test passes, commit the changes to git. Push your changes to Github.

Reply:
I think this goes back to what you said about the tests, in that the tests can't verify that the cache is actually doing anything, only that load() returns the correct result. In this case whether I remove this snippet or not the model will return the correct result.